src/BlazorWasmSdk/Tasks/GenerateBlazorWebAssemblyBootJson.cs

```diff
@@ -149,6 +152,12 @@ public void WriteBootJson(Stream output, string entryAssemblyName)
                             string.Equals(assetTraitValue, "native", StringComparison.OrdinalIgnoreCase))
                     {
                         Log.LogMessage(MessageImportance.Low, "Candidate '{0}' is defined as a native application resource.", resource.ItemSpec);
+                        if (string.Equals(fileName, "dotnet", StringComparison.OrdinalIgnoreCase) &&
+                            string.Equals(fileExtension, ".wasm", StringComparison.OrdinalIgnoreCase))
```

Comment:
Any reason we're case-insensitive here when most of the logic above isn't?

Reply:
We do case insensitive comparisons for files to keep things consistent between windows and other OSes

Reply:
I'm more asking about why the case sensitivity is different from the logic above.

Reply:
You mean compared to https://github.com/dotnet/sdk/pull/26966/files/a995950af430f86cfc1c55c16239a6e588549ece#diff-18a7293544774a63e6250aa358d64358829902f2f3d8b30dbe352b8c2178b63aL116? I think that is just a small bug from what I have seen in the rest of the file.